modules/light-clients/07-tendermint/migrations_test.go

```diff
@@ -0,0 +1,153 @@
+package tendermint_test
+
+import (
+	"time"
+
+	clienttypes "github.com/cosmos/ibc-go/v6/modules/core/02-client/types"
+	host "github.com/cosmos/ibc-go/v6/modules/core/24-host"
+	"github.com/cosmos/ibc-go/v6/modules/core/exported"
+	ibctm "github.com/cosmos/ibc-go/v6/modules/light-clients/07-tendermint"
+	ibctesting "github.com/cosmos/ibc-go/v6/testing"
+)
+
+// test pruning of multiple expired tendermint consensus states
+func (suite *TendermintTestSuite) TestPruneTendermintConsensusStates() {
+	// create multiple tendermint clients and a solo machine client
+	// the solo machine is used to verify this pruning function only modifies
+	// the tendermint store.
+
+	numTMClients := 3
+	paths := make([]*ibctesting.Path, numTMClients)
+
+	for i := 0; i < numTMClients; i++ {
+		path := ibctesting.NewPath(suite.chainA, suite.chainB)
+		suite.coordinator.SetupClients(path)
+
+		paths[i] = path
+	}
+
+	solomachine := ibctesting.NewSolomachine(suite.T(), suite.chainA.Codec, "06-solomachine-0", "testing", 1)
```

Comment:
is there a constant we can use for `"06-solomachine-0"`?

Reply:
I'm going to open an issue. I think we might be defining this string in multiple places, so I think it makes sense to fix up in a separate pr

Reply:
#2843